devlab/provision/generate_config.sh

```diff
@@ -34,9 +34,7 @@ while read key value
 do
     value=($value)
     value=${value[1]}
-    if [[ -n ${value} ]]; then
-      sed -i "s|<${key}>|${value}|g" ${result_config}
-    fi
+    sed -i "s|<${key}>|${value}|g" ${result_config}
```

Comment:
No! We must save it empty.
For example we have `prop` with default value `defVal`

Config file line | result
------------------|---------
prop=someVal | someVal
prop= | 
 | defVal

And if you delete this line from config file you will have default value.